trace_replay/trace_replay.cc

```diff
@@ -92,10 +92,8 @@ Status TracerHelper::DecodeTrace(const std::string& encoded_trace,
                                  Trace* trace) {
   assert(trace != nullptr);
   Slice enc_slice = Slice(encoded_trace);
-  if (!GetFixed64(&enc_slice, &trace->ts)) {
-    return Status::Incomplete("Decode trace string failed");
-  }
-  if (enc_slice.size() < kTraceTypeSize + kTracePayloadLengthSize) {
+  if (!GetFixed64(&enc_slice, &trace->ts) ||
+      enc_slice.size() < kTraceTypeSize + kTracePayloadLengthSize) {
```

Comment:
Should we instead report different error messages here?

Reply:
Actually, They're all failure about decode. I think it's fine . WDYT ~

Reply:
I prefer providing more information about the cause of failure when it's feasible and inexpensive.